resources/META-INF/studio-contribs.xml

```diff
@@ -15,6 +15,7 @@
   <extensions defaultExtensionNs="com.intellij">
     <androidStudioInitializer implementation="io.flutter.FlutterStudioInitializer"/>
     <postStartupActivity implementation="io.flutter.FlutterStudioStartupActivity"/>
+    <projectOpenProcessor implementation="io.flutter.project.FlutterStudioProjectOpenProcessor" order="after flutter"/>
```

Comment:
It's ok for these project open processors to not have ids?

Reply:
Hardly any extensions have id's. I should probably add one, just in case.